readthedocs/builds/models.py

```diff
@@ -1025,18 +1026,27 @@ def get_match_arg(self):
         )
         return match_arg or self.match_arg
 
-    def run(self, version, *args, **kwargs):
+    def run(self, version, register=True, **kwargs):
         """
         Run an action if `version` matches the rule.
 
         :type version: readthedocs.builds.models.Version
+        :param register: If ``True`` a register of the match is created
```

Comment:
I think we could remove this attribute. Do we have a case were we don't want to register them?

Reply:
When debugging

Reply:
If it's just for debugging, I'd remove it so we don't have to maintain code that it's not needed.